content/panel_release_1.3.ipynb

```diff
@@ -0,0 +1,1970 @@
+{
```

Comment:
<p>Can we add an interactive demo of <code>TextAreaInput(placeholder="Enter new lines to see this grow!", auto_grow=True, max_rows=4)</code> too? I think it's quite relevant to ChatInterface.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>